src/select.rs

```diff
@@ -77,24 +88,53 @@ where
         }
         <Self as PromptInteraction<T>>::interact(self)
     }
+
+    /// Enable the filter mode, press / to activate
+    pub fn enable_filter(mut self) -> Self {
+        self.enable_filter_mode = true;
+        self
+    }
 }
 
 impl<T: Clone> PromptInteraction<T> for Select<T> {
     fn on(&mut self, event: &Event) -> State<T> {
         let Event::Key(key) = event;
 
         match key {
-            Key::ArrowLeft | Key::ArrowUp => {
+            Key::ArrowUp => {
                 if self.cursor > 0 {
                     self.cursor -= 1;
                 }
             }
-            Key::ArrowRight | Key::ArrowDown => {
+            Key::ArrowDown => {
                 if self.cursor < self.items.len() - 1 {
                     self.cursor += 1;
                 }
             }
-            Key::Enter => return State::Submit(self.items[self.cursor].value.clone()),
+            Key::ArrowRight => {
```

Comment:
Seems like, in the "filter mode" ("fuzzing mode"?) the right and left arrows should move the cursor of the input filter left and right.

Reply:
I am not sure if I understood this comment well

When the user is in "fuzzing mode" I disabled the changing of the selected input when pressing left and right keys in order to move into the input (filter var atm) without impacting the current selected item
The normal behavior when being outside of the "fuzzing mode" remains the same as before

Do you think I should keep the same behavior while being in "fuzzing mode" ?

Reply:
@antoinePutmans I was thinking about how to make less code duplication and also about the cursor behavior: there is an input field in the fuzzing mode, thus the cursor appears, and I guessed that "left" and "right" would move the cursor throughout the input text. However, now I think it complicates everything, so... let's keep them disabled, no changes are needed.

Reply:
I agree with the original idea here, that left/right arrows should navigate the **filter text input** and simply for user-friendliness -- but I'm not against this being chalked up as a separate issue for future impl so that this PR can move forward :)